server/app.js

```diff
@@ -17,7 +17,7 @@ const questions = require('./routes/questions');
 const { json, urlencoded } = express;
 
 var app = express();
-app.use(cors());
+app.use(cors({ origin: 'http://localhost:3000', credentials: true }));
```

Comment:
We shouldn't need to set up cors if we are using the proxy - and this will be hard to deal with when we deploy

Reply:
So I was getting CORS errors because we are fetching from :3000 over to :3001.  sameSite: 'lax' is okay, for sending over the cookie?  I had many hours of issues because of cross-site origins.